src/components/Navabar/Navbar.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {render, screen, fireEvent} from '@testing-library/react';
+import Navbar from './Navbar.component';
+
+describe('<Navbar />', ()=>{
+    const mockHandler = jest.fn();
+    test('Render Navbar component', ()=>{
+        render(<Navbar />)
+    });
+    test('renders by Search', () => {
+        render(<Navbar />);
+        const search = screen.getByPlaceholderText(/Search/i);
+        expect(search).toBeInTheDocument();
+    });
+    test('renders by Dark Mode', () => {
+        render(<Navbar />);
+        const darkMode = screen.getByText(/Dark Mode/i);
+        expect(darkMode).toBeInTheDocument();
+    });
+});
```

Comment:
Nice test cases definitions, 

Probably the tests descriptions could be like "render the search input", "render the dark mode text", for example 😄

Reply:
I recommend you to use the `beforeEach` Jest function in order to execute the `render(<Navbar />)` sentence, so you can remove it from the start of each test.

This is util when you have the same initial "setup" on the test cases, on this case, the `render(<Navbar />)` is the setup for each one